modelconverter/packages/rvc2/exporter.py

```diff
@@ -109,25 +109,50 @@ def _export_openvino_ir(self) -> Path:
                 reverse_only=True,
             )
             for inp in self.inputs.values():
-                if inp.mean_values is not None and inp.reverse_input_channels:
+                if (
+                    inp.mean_values is not None
+                    and inp.encoding.from_ != inp.encoding.to
+                ):
                     inp.mean_values = inp.mean_values[::-1]
-                if inp.scale_values is not None and inp.reverse_input_channels:
+                if (
+                    inp.scale_values is not None
+                    and inp.encoding.from_ != inp.encoding.to
+                ):
                     inp.scale_values = inp.scale_values[::-1]
-                inp.reverse_input_channels = False
+                inp.encoding.from_ = Encoding.BGR
+                inp.encoding.to = Encoding.BGR
```

Comment:
This was added to be aligned with the previous implementation where `inp.reverse_input_channels = False` inside this if statement.
This is a special case where the model has >1 inputs and the inputs have different encodings (so one needs reverse of the channels and the other one does not). Since the `--reverse_input_channels` in the OpenVino is a universal one (e.g. applies to all inputs) what we are doing here is that we modify the onnx to add the necessary extra nodes to the inputs where channels need to be reversed and we make sure the `--reverse_input_channels` will not be set. To make that happen and due to this check [here](https://github.com/luxonis/modelconverter/blob/cbfdd2326aee31c7ad05ebea7cd95a38010e9ab3/modelconverter/packages/rvc2/exporter.py#L150-L155) we need to make sure that for all inputs the `encoding.from` and `encoding.to` will be the same and since the DAI default is BGR we explicitly set this to BGR.
Does this clear things out? Does this make sense?